libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -246,6 +247,8 @@ void stopNavigation() {
     navigation.unregisterRouteProgressObserver(navigationViewVm);
     navigation.unregisterLocationObserver(navigationViewVm);
     navigation.unregisterVoiceInstructionsObserver(voiceInstructionsObserver);
+    navigation.unregisterOffRouteObserver(offRouteListener);
```

Comment:
Can we remove this line? We have it in other PR. This will cause merge conflicts.

Reply:
Could we merge here to unlock other work? This should be a simple fix (or maybe git will even figure it out by itself).

Reply:
given it's a simple and needed fix. we might want to merge it asap. I would assume the other PR is the one to create the ExampleActivity and that PR will need more time for reviewing before merge.